src/main/java/com/st/utopia/counter/controller/CounterCancellationController.java

```diff
@@ -0,0 +1,48 @@
+package com.st.utopia.counter.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+import com.st.utopia.counter.entity.Ticket;
+
+/**
+ * Controller for Counter API
+ * 
+ * @author Al-amine AHMED MOUSSA
+ */
+
+@RestController
+@CrossOrigin
+@RequestMapping("/counter")
+public class CounterCancellationController {
```

Comment:
The routes in this file are not part of the API we documented, and seem to duplicate functionality you provide elsewhere.